Tests/SonarQube.Bootstrapper.Tests/ArgumentProcessorTests.cs

```diff
@@ -257,9 +256,72 @@ public void ArgProc_BeginAndEndVerbs()
             logger.AssertSingleErrorExists("begin", "end");
         }
 
-        #endregion
+        [TestMethod]
+        public void ArgProc_Verbosity()
```

Comment:
This doesn't seem like the right place to test all of these cases. Most of these should be in the unit tests for the VerbosityCalculator class itself  i.e. as direct tests for "public static LoggerVerbosity ComputeVerbosity(IAnalysisPropertyProvider properties, ILogger logger)".

The unit tests for the ArgumentProcessor should focusing mainly on the behaviour of the ArgumentProcessor itself.